src/main/resources/OSGI-INF/auth-contrib.xml

```diff
@@ -0,0 +1,38 @@
+<component name="org.nuxeo.wopi.auth">
+
+  <extension target="org.nuxeo.ecm.platform.ui.web.auth.service.PluggableAuthenticationService"
+    point="preFilter">
+
+    <preFilter name="wopi" order="0" class="org.nuxeo.ecm.platform.ui.web.auth.service.WOPIAuthFilter" />
+
+  </extension>
+
+  <extension target="org.nuxeo.ecm.platform.ui.web.auth.service.PluggableAuthenticationService"
+    point="specificChains">
+
+    <specificAuthenticationChain name="wopi" handlePrompt="false">
```

Comment:
Do you need this? `TOKEN_AUTH` is already part of the default WebEngine auth plugins.

Reply:
It will be removed anyway when migrating to JWT auth.

Reply:
Probably not, but we'll switch to `JWTAuthenticator` anyways.

Reply:
Ok